types/got/index.d.ts

```diff
@@ -170,6 +172,11 @@ declare namespace got {
      */
     type BeforeRetryHook<Options> = (options: Options, error: GotError, retryCount: number) => any;
 
+    /**
+     * @param error Request or parse error.
```

Comment:
nit: I think tsdoc can be omitted too, because error can be not only request or parse error. Anyway it's obvious.

Reply:
Yeah, I added it to be consistent with the the types above hook types. I'm happy to remove it though.